src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -277,29 +287,26 @@ def file_issue(testcase,
     update_issue_impact_labels(testcase, issue)
 
   # Add additional labels from the job definition and fuzzer.
-  additional_labels = data_handler.get_additional_values_for_variable(
-      'AUTOMATIC_LABELS', testcase.job_type, testcase.fuzzer_name)
+  additional_labels = _get_additional_values(testcase, 'AUTOMATIC_LABELS')
   for label in additional_labels:
     issue.labels.add(label)
 
   # Add additional components from the job definition and fuzzer.
-  automatic_components = data_handler.get_additional_values_for_variable(
-      'AUTOMATIC_COMPONENTS', testcase.job_type, testcase.fuzzer_name)
+  automatic_components = _get_additional_values(testcase,
+                                                'AUTOMATIC_COMPONENTS')
   for component in automatic_components:
     issue.components.add(component)
 
   # Add issue assignee from the job definition and fuzzer.
-  automatic_assignee = data_handler.get_additional_values_for_variable(
-      'AUTOMATIC_ASSIGNEE', testcase.job_type, testcase.fuzzer_name)
+  automatic_assignee = _get_additional_values(testcase, 'AUTOMATIC_ASSIGNEE')
```

Comment:
We select automatic_assignee[0] below, so this change looks unnecessary ?

Reply:
Just for consistency -- the code looks shorter that way anyway :)